nemo_skills/inference/prompt/openai_chat.yaml

```diff
@@ -0,0 +1,7 @@
+delimiter: "\n\n\n\n\n\n"
```

Comment:
How about we make our code more general to handle cases like this. E.g. one way to do that is to add a new "system" parameter that's reserved for cases when system message is well defined. And to keep supporting our code where system is just a prefix, we can add a new "prompt_template" parameter that will be something like this:

```
prompt_template: "System: {system}\n\n{examples}"
```
where examples is what we currently compose in utils that uses `template` and `delimiter`. This way we can customize prefix/suffix of prompt with more flexibility and also get to know an actual system message in case it needs to be used parsed out like when we use openai or when we prepare nemo chat datasets.

Maybe also rename the current `template` to `example_template` to make it clearer